resources/views/emails/messages/plain_text.blade.php

```diff
@@ -0,0 +1,6 @@
+Dear {!! isset($recipientName) ? $recipientName : $recipient->name !!},
+
+{{  strip_tags($body) }}
```

Comment:
Yes, the tags are gone and it displays nicely, helped by the fact that the new lines in the email bodies are conserved. 99% of the time the user will never see these, it's generally used for excerpts and spam detection